cpp/include/DataStorm/DataStorm.h

```diff
@@ -1424,19 +1424,19 @@ namespace DataStorm
     }
 
     template<typename Key, typename Value, typename UpdateTag>
-    template<typename SFC>
+    template<typename SampleFilterCriteria>
```

Comment:
Not using the same names as in the declaration (earlier in the same file) confuses doxygen.